webapp/src/Entity/Rejudging.php

```diff
@@ -118,6 +118,26 @@ class Rejudging
      */
     private $autoApply = true;
 
+    /**
+     * @var int
+     *
+     * @ORM\Column(type="integer", name="`repeat`",
+     *     options={"comment"="Number of times this rejudging will be repeated.",
+     *              "unsigned"=true,"default"="NULL"},
+     *     nullable=true)
+     */
+    private $repeat;
+
+    /**
+     * @var int
+     *
+     * @ORM\Column(type="integer", name="repeat_rejudgingid",
+     *     options={"comment"="In case repeat is set, this will mark the first rejudgingid.",
+     *              "unsigned"=true,"default"="NULL"},
+     *     nullable=true)
+     */
+    private $repeat_rejudgingid;
```

Comment:
Shouldn't we change this to an actual entity relation? Then a foreign key will also be created.

Reply:
How? I wanted to copy what we do in other places and don't see any difference to e.g. https://github.com/DOMjudge/domjudge/blob/master/webapp/src/Entity/JudgingRun.php#L47

Reply:
That also has this: https://github.com/DOMjudge/domjudge/blob/master/webapp/src/Entity/JudgingRun.php#L92

We can even leave out the id field itself I believe, it’s not really used and sometimes gives issues if you also have a linked entity.

Reply:
Is that what you had in mind?

Reply:
Beautiful. We could remove the other field but if this works ship it

Reply:
We have this in other places as well so I wonder what's best practice.

Reply:
(we don't need to load any other data than the id itself if it matters)

Reply:
I think best practice is to not have both but we indeed have it a lot. Also you can call something like $em->getreference to not load the whole entity iirc. But as said let’s keep it consistent for now. Maybe we can do a separate PR where we remove all these columns